src/main/java/org/opentripplanner/netex/mapping/StationMapper.java

```diff
@@ -48,7 +48,9 @@ Station map(StopPlace stopPlace) {
       name,
       WgsCoordinateMapper.mapToDomain(stopPlace.getCentroid()),
       null,
-      stopPlace.getDescription() != null ? stopPlace.getDescription().getValue() : null,
+      stopPlace.getDescription() != null
```

Comment:
You can use `NonLocalizedString#ofNullable` for these

Reply:
StopPlace’s description may be null, at least in tests.  If we only use `NonLocalizedString#OfNullable` here, it will cause nullpointer, since stopPlace.getDescription() returns null.

Reply:
The `NonLocalizedString#ofNullable` is null-safe...

Reply:
Yes that is true. But I was meaning that because `StopPlace.getDescription()` may be null,  when you use `NonLocalizedString.OfNullable(stoPlace.getDescription.getValue()) `it will cause nullpointer.

Reply:
One possibility is to use `Optional.ofNullable(stopPlace.getDescription()).map(MultilingualString::getValue).map(NonLocalizedString::new).orElse(null)`

Or even create `NonLocalizedString#ofOptional(Optional<String>)` for cases like these

Reply:
Sorry I missed the nesting, I do not think Optional should be used here. So the code is fine.

Reply:
A solution here is to create a MultilingualStringMapper in the Netex mappers with a static method - but I do not expect this PR to do that.